core/src/main/java/org/polypheny/db/algebra/rules/ScanRule.java

```diff
@@ -40,15 +43,19 @@ public class ScanRule extends AlgOptRule {
      * @param algBuilderFactory Builder for relational expressions
      */
     public ScanRule( AlgBuilderFactory algBuilderFactory ) {
-        super( operand( LogicalScan.class, any() ), algBuilderFactory, null );
+        super( operandJ( Scan.class, Convention.NONE, r -> true, any() ), algBuilderFactory, ScanRule.class.getSimpleName() );
```

Comment:
operandJ is not new, but still not a "speaking" (i.e., self-explanatory) name. I assume the J stands for Java since the operandJ methods use `java.util.function.Predicate` instead of Guva used in operand. However, why not get rid of Guava methods entirely? Or is there another reason for the naming?

Reply:
Agreed, the implementations using guave are unnecessary at this point, refactored and removed guave.